src/wram.asm

```diff
@@ -1388,7 +1412,8 @@ wcea1:: ; cea1
 
 	ds $1
 
-wcea3:: ; cea3
+; used to blink the cursor in duel menu
+wDuelCursorBlinkCounter:: ; cea3
```

Comment:
Since this only used in the check menu, I'd rename it to ``wCheckMenuCursorBlinkCounter``. Same thing as I suggested in #52 :)